src/checks/stylelint.ts

```diff
@@ -0,0 +1,65 @@
+import { execaCommand } from 'execa';
+import { LintResult } from 'stylelint';
+import { projectPackageJson } from '../util/packageJson';
+
+const projectDependencies = projectPackageJson.dependencies || {};
+
+export const supportedFileExtensions = [
+  'css',
+  'sass',
+  'scss',
+  'less',
+  'js',
+  'tsx',
+  'jsx',
+];
+
+export const title = 'Stylelint';
+
+export default async function stylelintCheck() {
+  // Continue only if project is Next.js or UpLeveled React App
+  if (
+    !(
+      '@upleveled/react-scripts' in projectDependencies ||
+      'next' in projectDependencies
+    )
+  ) {
+    return;
+  }
+
+  try {
+    await execaCommand(
+      `yarn stylelint **/*.{${supportedFileExtensions.join(
```

Comment:
these commands need to be updated to `pnpm`, v2 uses `pnpm`:

- https://github.com/upleveled/preflight/releases/tag/v2.0.0

Reply:
https://github.com/upleveled/preflight/pull/377/commits/d0d08f4559cd2474cb7b223ab1dc1824c2a0d07b